### ISSUE TITLE
CSS and SCSS files in the same folder are causing problem when building with webpack

### ISSUE THREAD

**Issue by luizs81** [jossef/material-design-icons-iconfont#11]
```
Module build failed: 
@import "material-design-icons-iconfont/dist/material-design-icons";
        ^
It's not clear which file to import for '@import "material-design-icons-iconfont/dist/material-design-icons"'.
Candidates:
material-design-icons-iconfont/dist/material-design-icons.scss
material-design-icons-iconfont/dist/material-design-icons.css
Please delete or rename all but one of these files.
```

**Comment by jossef** [comment]
sorry it took me long to reply, 
in your case, you need to specify the file's extension. e.g.

    @import "material-design-icons-iconfont/dist/material-design-icons.scss";

having said that, issue #10 requests to move the `scss` file to somewhere else
will be moved shortly

**Comment by jossef** [comment]
fixed in 4.0.1 release
thanks!
